data/layout/telescope_positions-North-1LST.ecsv

```diff
@@ -1,55 +1,50 @@
-# %ECSV 0.9
+# %ECSV 1.0
 # ---
 # datatype:
-# - {name: telescope_name, datatype: string}
-# - {name: pos_x, unit: m, datatype: float64}
-# - {name: pos_y, unit: m, datatype: float64}
-# - {name: pos_z, unit: m, datatype: float64}
-# - {name: lat, unit: deg, datatype: float64}
-# - {name: lon, unit: deg, datatype: float64}
-# - {name: utm_east, unit: m, datatype: float64}
-# - {name: utm_north, unit: m, datatype: float64}
-# - {name: alt, unit: m, datatype: float64}
+# - {name: telescope_name, datatype: string, description: common telescope name}
+# - {name: pos_x, unit: m, datatype: float32, description: telescope position towards North relative to MST15 (central MST)}
+# - {name: pos_y, unit: m, datatype: float32, description: telescope position towards West relative to MST15 (central MST)}
+# - {name: pos_z, unit: m, datatype: float32, description: telescope position upwards relative to NNN}
 # meta:
+#   data_type: positionfile
+#   description: "telescope positions for CTA North (La Palma)"
 #   EPSG: 32628
 #   center_alt: !astropy.units.Quantity
 #     unit: &id001 !astropy.units.Unit {unit: m}
-#     value: 2150.0
+#     value: 2177
 #   center_easting: !astropy.units.Quantity
 #     unit: *id001
-#     value: 217654.06411838497
+#     value: 217611.227
 #   center_lat: !astropy.units.Quantity
 #     unit: &id002 !astropy.units.Unit {unit: deg}
-#     value: 28.762158
+#     value: 28.7621661
 #   center_lon: !astropy.units.Quantity
 #     unit: *id002
-#     value: -17.891571
+#     value: -17.8920302
 #   center_northing: !astropy.units.Quantity
 #     unit: *id001
-#     value: 3185065.2899643113
+#     value: 3185066.278
 #   corsika_obs_level: !astropy.units.Quantity
 #     unit: *id001
-#     value: 2158.0
+#     value: 2158
 #   corsika_sphere_center:
 #     LST: !astropy.units.Quantity
 #       unit: *id001
 #       value: 16.0
 #     MST: !astropy.units.Quantity
 #       unit: *id001
 #       value: 9.0
-#     SST: !astropy.units.Quantity
-#       unit: *id001
-#       value: 3.25
 #   corsika_sphere_radius:
 #     LST: !astropy.units.Quantity
 #       unit: *id001
 #       value: 12.5
 #     MST: !astropy.units.Quantity
 #       unit: *id001
 #       value: 9.6
-#     SST: !astropy.units.Quantity
-#       unit: *id001
-#       value: 3.0
+#   comments: |
+#       version 0.1
+#       UTM Zone 28
+#       prod5
```

Comment:
I am not sure these values are really for Prod5 since they changed since. Also, we shouldn't list the version here, it should be in the parameter pointing to this file.